#include <bits/stdc++.h>
using namespace std;


int main() {
	try {
		int a = 4 / 0;
	} 
	catch (exception& e) {
		cout << "Exception";
	}
}
